config-model/src/test/java/com/yahoo/config/model/provision/ModelProvisioningTest.java

```diff
@@ -1313,6 +1314,28 @@ public void testThatStandaloneSyntaxWorksOnHostedVespa() {
         assertEquals(1, model.getContainerClusters().size());
     }
 
+    @Test
+    public void testThatStandaloneSyntaxOnHostedVespaRequiresDefaultPort() {
```

Comment:
Consider removing "hosted Vespa" from test name as this test does not use a hosted specific model.

Reply:
True, but it is evaluated in an environment set up as hosted Vespa (otherwise the test would fail).

Reply:
Nvm, I did not know that `VespaModelTester` default to creating a hosted specific model.